### PR TITLE
Usage documentation

### PR DESCRIPTION
**Pull request by bendialpad**
Adds first draft of overview and usage documentation. I need to add some examples before merging this. 

Ironically the formatting looks terrible, I feel like i'm probably missing some styles here. Any help making this look better is appreciated!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,35 @@ Dialpad Design is the home for Dialtone, Dialpad's design system. It includes th
 
 **This is under construction.**
 
+## Running Locally
```
Comment (bendialpad) [inline README.md]: This is what I did but i'm not convinced it was the best way or works particularly well (I had to remove the inline-svg plugin locally as it wouldn't build). So any updates here would be good.

File: docs/usage/overview.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+A design system is a collection of common styles, components and associated usage documentation. These styles and components will be usable anywhere within the Dialpad and UC projects.
+</p>
+
+<p>
+The aim of Dialtone is to help ensure consistency of design across all products whilst also reducing time spent building new functionality by providing reusable classes and components. 
+</p>
+
+<h2>Key Contributors & Responsibilities</h2>
+
+<ul>
```
Comment (bendialpad) [inline docs/usage/overview.html]: Please let me know if I missed anyone here or if you'd like a better description. Maybe we shouldn't even have this bit, I just wanted to document people involved so if anyone has any questions etc they know who to go to.

File: docs/usage/usage.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+<p>
+	Dialtone can be used in both <a href="#dialpad">Dialpad</a> and <a href="#uc">Uberconference</a>. Below you can find general examples of how to use Dialtone in each project. You can find specific information on the styles available and how to use them using the navigation on the left.
+</p>
+
+<a name="dialpad"><h2>Using Dialtone in Dialpad</h2></a>
+
+<h3>Vue</h3>
+
+<p>
+	TODO: Example coming
```
Comment (bendialpad) [inline docs/usage/usage.html]: Will add tomorrow.

File: docs/usage/usage.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+</p>
+
+<h3>Backbone</h3>
+
+<p>
+	TODO: Example coming
```
Comment (bendialpad) [inline docs/usage/usage.html]: Ditto.

File: docs/usage/usage.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+</p>
+
+<p>
+	In <span class="d-ff-mono">Backbone</span>  we are not using <span class="d-ff-mono">border-box</span> by default. Because Dialtone expects this, anytime we wish to use Dialtone styles in Backbone we must ensure to apply the <span class="d-ff-mono">box-sizing: border-box</span> style to all affected elements.
+</p>
+
+<a name="uc"><h2>Using Dialtone in UberConference</h2></a>
+
+<p>
+	Coming soon.
```
Comment (bendialpad) [inline docs/usage/usage.html]: @freddialpad if you have something I can put here that would be great.

File: docs/usage/best-practices.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+layout: page
+title: Best Practices
+description: TODO
```
Comment (bendialpad) [inline docs/usage/best-practices.html]: Would appreciate some input here. Maybe we can have a quick chat about this tomorrow.

File: docs/usage/contributing.html [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+layout: page
+title: Contributing to Dialtone
+description: TODO
```
Comment (bendialpad) [inline docs/usage/contributing.html]: For now I think we decided on 1. figuring out if something is generic enough that it should be in Dialtone and 2. to submit a PR to github following the existing conventions. Are there any more specifics to add here?

File: docs/usage/overview.html [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+</p>
+
+<p>
+	A design system is a collection of common styles, components and associated usage documentation. These styles and components will be usable anywhere within the Dialpad and UC projects.
+</p>
+
+<p>
+	The aim of Dialtone is to help ensure consistency of design across all products whilst also reducing time spent building new functionality by providing reusable classes and components. 
+</p>
+
```
Comment (bendialpad) [inline docs/usage/overview.html]: I need to add a paragraph here explaining only the styling part is available for now. Probably should add that in usage too.

File: README.md (hunk 1 above)
Comment (hynes-dialpad) [reply README.md]: @bendialpad I've updated the README. Also I removed `jekyll-inline-svg` from the project. I wasn't being used anyway.

**Comment by hynes-dialpad** [comment]
@bendialpad I've gone through and update the documentation to be correctly formatted. Also I made some minor copy changes here and there. Thanks for putting this together! This looks good enough for me to push for now and we can continue to add to it over the next couple of weeks.

**Review by hynes-dialpad — APPROVED** [review]
LGTM! It's not complete, but it roughs out the areas and we can fill in the holes over the coming weeks.

File: docs/getting-started/overview.html [hunk 9]
```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Overview
+description: The 4-1-1 on Dialtone, it's goals, and how to use it.
```
Comment (dbecherdialpad) [inline docs/getting-started/overview.html]: forgive me, for I am going to be That Guy: *its

Comment (bendialpad) [reply docs/getting-started/overview.html]: I also don't really know what 4-1-1 means (obviously I can guess :P). Might be good to rephrase this to something everyone will understand.

File: docs/getting-started/overview.html [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+---
+<div class="dialtone-section--xxs">
+    <h2 class="title__subsection">What is Dialtone?</h2>
+    <p class="copy__normal">Dialtone is the new design system for Dialpad and UberConference.</p>
```
Comment (bendialpad) [inline docs/getting-started/overview.html:8]: Any reason you removed the link to the webpage explaining what a design system is? I think it's important to link to an external resource that explains the concept in more detail for those wishing to get a greater understanding. If you have a better resource than the one I chose then that'll be great!

File: docs/getting-started/overview.html [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+</div>
+<div class="dialtone-section--xxs">
+    <h2 class="title__subsection">Key Contributors & Responsibilities</h2>
+    <ul class="copy__normal">
+        <li><strong>Joshua Hynes</strong> - Creator of Dialtone, CSS & Design decisions</li>
+        <li><strong>Robert Hurst</strong> - Core product engineering contributor, common components</li>
+        <li><strong>David Becher</strong> - Product engineering contributor</li>
+        <li><strong>Ben Holmes</strong> - Product engineering contributor</li>
+        <li><strong>Trinh Nguyen</strong> - Project Manager</li>
+        <li><strong>Spencer Holtaway</strong> - Design Manager</li>
```
Comment (bendialpad) [inline docs/getting-started/overview.html:28]: Need to add Steve

File: docs/getting-started/overview.html (hunk 9 above)
Comment (hynes-dialpad) [reply docs/getting-started/overview.html]: 4-1-1 is a phone number you can call for directory assistance in the US and Canada. 😄 

https://en.wikipedia.org/wiki/4-1-1

File: docs/getting-started/overview.html (hunk 10 above)
Comment (hynes-dialpad) [reply docs/getting-started/overview.html:8]: Ok. I can add that link back in. I have a lot of links I could provide here. 😄

**Review by dbecherdialpad — APPROVED** [review]

File: docs/usage/usage.html (hunk 5 above)
Comment (superfreddialpad) [reply docs/usage/usage.html]: I see this has already landed, but I'm just getting back from vacation. I'll add UC docs once we're all the way switched over to less.
